presto-hive/src/test/java/com/facebook/presto/hive/AbstractTestHiveClient.java

```diff
@@ -2535,6 +2538,124 @@ public void testCreateTableUnsupportedType()
         }
     }
 
+    @Test
+    public void testCreateEmptyBucketedTemporaryTable()
```

Comment:
Looks like in `AbstractTestHiveClient`, the name convention is something like `testTableCreation`, `testBucketedTableCreation`. 

I am OK with this name convention :). Just want to double check if you did this with intention :)

Reply:
The naming is rather inconsistent. There are `testMetadataDelete`, `testInsertUnsupportedWriteType`, `testUpdatePartitionColumnStatistics`, etc. Let's keep it as is.